### ISSUE TITLE
[rt.openssl.org #3913] Add a way to application to know a minimum DH size allowed by the client

### ISSUE THREAD

**Issue by richsalz** [openssl/openssl#2370]
Migrated from [rt.openssl.org#3913](https://rt.openssl.org/Ticket/Display.html?id=3913) (status was 'new')

Requestors:
* tmraz@redhat.com

From tmraz@redhat.com on 2015-06-17 10:38:45:
```
The current minimum DH size allowed by the client is 768 bits which is a
hardcoded constant. It would be nice if the constant was at least
#define in public headers or even better if there was an API to query
various minimum and maximum bit sizes that are checked in the library
such as the maximum supported key lengths, etc.

-- 
Tomas Mraz
No matter how far down the wrong road you've gone, turn back.
                                              Turkish proverb
(You'll never know whether the road is wrong though.)


_______________________________________________
openssl-bugs-mod mailing list
openssl-bugs-mod@openssl.org
https://mta.openssl.org/mailman/listinfo/openssl-bugs-mod
```

**Comment by nhorman** [comment]
Marking as inactive to be closed at the end of 3.4 dev barring further input

**Comment by richsalz** [comment]
@t8m, you're not gonna add the public define? :)
